test/resources/e2etest.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+cd test/resources
+kubectl apply -f addon1.yaml
+cat ../../deploy/service_account.yaml divider.txt ../../deploy/role.yaml divider.txt ../../deploy/role_binding.yaml divider.txt ../../deploy/operator.yaml mount.txt > namespaced.yaml
+cd -
+operator-sdk test local ./test/e2e --namespace default --namespaced-manifest test/resources/namespaced.yaml  --global-manifest test/resources/operator_v1alpha1_config_cr.yaml  --image $1
```

Comment:
should be `--namespace ""` as `config.operator.tekton.dev` is a clusterwide crd

Reply:
i will add this test as a make target so that the tests can be run with the image build buy the CI, instead of always using the image from `deploy/operator.yaml`